build.gradle

```diff
@@ -738,3 +739,11 @@ task pusher(type: KotlinBuildPusher){
     overrideConfigurationId = System.getenv("TEAMCITY_OVERRIDE_NATIVE_ID") ?: "Kotlin_KotlinDev_DeployMavenArtifacts_OverrideNative"
     token = project.findProperty("teamcityBearToken") ?: System.getenv("TEAMCITY_BEAR_TOKEN")
 }
+
+task format {
+    dependsOn ":runtime:format"
+}
+
+task formatCheck {
```

Comment:
This should be run during a precommit check as a part of some existing build or as a brand new build (which can be run on any platform, since there isn't anything platform-specific in this style check)

Reply:
@PavelPunegov which way is better in your opinion: make this check a part of an existing build, or create another build?

Reply:
I think it could be done as a part of sources checkout job. 
I see two scenarios here:
1. We could implement some reporting that will be collected and shown. Some kind of warnings.
2. Fail on checkout and fail the whole job chain line. This one enforces usage of format rules, but may add some undesired failures where the code is not formatted yet or under the discussion whether formatter correct or should be changed, and the code itself doesn't build and tested.

Reply:
I don't think warnings will do the job: one can easily forget to check them. Is there no simple way to fail a build, while allowing the build chain to continue?
I do agree that failing the entire chain because of formatting is at the very least annoying.

Reply:
AFAIK, adding this check as additional "Precommit" configuration dependency should do the job.